modules/ibi-create-iso-for-seed-image.adoc

```diff
@@ -0,0 +1,93 @@
+// Module included in the following assemblies:
+//
+// * edge_computing/ibi-image-based-install.adoc 
+
+:_content-type: PROCEDURE
+[id="embed-sno-to-live-iso_{context}"]
+= Embedding the installation artifacts in a live installation ISO
+
+You can embed your {sno} seed image URL, and other installation artifacts, in a live installation ISO by using the `ib-cli` CLI tool. Extract the `ib-cli` CLI tool binary from the {lcao}. 
+
+For further information about the command line arguments for the `ib-cli` CLI tool, see the _ib-cli CLI tool reference_ section.
+
+.Prerequisites
+* You generated a seed image from a {sno} seed cluster.
+* You installed Podman.
+* The target host has network access to the seed image URL and all other installation artifacts.
+
+.Procedure
+
+. Extract the `ib-cli` CLI tool from the {lcao}:
+
+.. Create a temporary container for the {lcao} image by running the following command:
++
+[source,terminal]
+----
+$ podman create --name temp_container quay.io/openshift-kni/lifecycle-agent-operator:latest
```

Comment:
lets use 4.16 and not latest as latest will constantly change

Reply:
@eranco74 maybe we should provide and option to get command from seed and not from lca?